source/MaterialXTest/GenOgsfx.cpp

```diff
@@ -85,34 +86,19 @@ TEST_CASE("OGSFX Unique Names", "[genogsfx]")
     GenShaderUtil::testUniqueNames(context, mx::HW::FX_STAGE);
 }
 
-class OGSFXGenCodeGenerationTester : public GenShaderUtil::ShaderGeneratorTester
+class OGSFXGenCodeGenerationTester : public GLSLGenCodeGenerationTester
```

Comment:
Nitpick: Same as above maybe name this class `OgsFxShaderGeneratorTester`

Reply:
Will do.